### DISCUSSION TITLE
When two windwos are next to each other, one is very slow if the screen has scaled resolution.

### DISCUSSION THREAD

**Discussion by dd3f** [i3/i3#4588] [Q&A]
Hi! 
After a recent update of the entire system (arch linux), when I have two windows next to each other in the external screen (for example Firefox or even alacritty), any operation on the window on the right is very very slow making the second screen completely unusable. With "any operation" I mean scroll a web page or even write on the alacritty terminal.

I find that if the screen has no scaled resolution  or a windows uses the entire screen, everything works okay. 
 However, since the laptop screen has a HiDPI screen, everything looks huge without scaling the external monitor.
Oddly enough, if the right window  is a terminal like terminator, it works completely fine. So i have no idea how to debug it or correct it. 

To give more details, I set my external monitor resolution with
`xrandr  --output DP-3 --mode 2560x1080 --scale 2x2 --right-of eDP-1 --rate 60.00`
and  `xrandr --listmonitors` gives:
```
Monitors: 2
 0: +*eDP-1 3840/344x2160/194+0+0  eDP-1
 1: +DP-3 5120/798x2160/334+3840+0  DP-3
```
my laptop is an xps 3570

Any suggestion?

**Comment by Airblader** [comment]
A wild guess is that this is another thing with RandR 1.5. Can you try adding `disable_randr15 yes` to your config and restarting i3?

**Reply by dd3f** [reply]
Thank you for the suggestion. I included but the results did not change. I am afraid it is not a problem with i3. I installed xfce and I get similar issue: I open alacritty and I move it around. Whenever the window is on the right side of my external screen (on the right of the laptop) then writing get slowed down. When scaled by two that should mean that the issue starts when the windows is on the right of the (virtual) pixel 6400.
```
Screen 0: minimum 320 x 200, current 8960 x 2160, maximum 16384 x 16384
eDP-1 connected primary 3840x2160+0+0 (normal left inverted right x axis y axis) 344mm x 194mm
     3840x2160     60.00*+  59.98    59.97
DP-3 connected 5120x2160+3840+0 (normal left inverted right x axis y axis) 798mm x 334mm 
     2560x1080     59.98*+  74.99    50.00
```

Any idea?

**Reply by Airblader** [reply]
At this point I would suspect the graphics driver stack, but this is leaving my area of knowledge.
